### PR TITLE
Fix typo in test_catkin_virtualenv_py3/CMakelist.txt

### PR DESCRIPTION
**Pull request by knorth55**
- fix typo

### CODE REVIEW

File: catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake [hunk 1]
```diff
@@ -58,8 +58,7 @@ function(catkin_generate_virtualenv)
     OUTPUT_STRIP_TRAILING_WHITESPACE
   )
 
-  set(PYTHON_VERSION_MAJOR ${ARG_PYTHON_VERSION_MAJOR})
```
Comment (paulbovbel) [inline catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake]: not sure what the goal of this change is, but it breaks the nosetest logic further down.

File: test_catkin_virtualenv_py3/CMakeLists.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@ find_package(catkin REQUIRED COMPONENTS catkin_virtualenv)
 catkin_package()
 
 catkin_generate_virtualenv(
-  PYTHON_MAJOR_VERSION 3
+  PYTHON_VERSION_MAJOR 3
```
Comment (paulbovbel) [inline test_catkin_virtualenv_py3/CMakeLists.txt]: good catch!

File: catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake (hunk 1 above)
Comment (knorth55) [reply catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake]: does it cause test failures on travis?
it seems `PYTHON_VERSION_MAJOR` is no need to set.

Comment (knorth55) [reply catkin_virtualenv/cmake/catkin_generate_virtualenv.cmake]: kindly ping @paulbovbel

**Comment by knorth55** [comment]
Updated.

**Pull request merged**